### PR TITLE
Update freesmug-chromium from 80.0.3987.116 to 81.0.4044.122

### PR DESCRIPTION
**Pull request by fohria**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by miccal** [comment]
@fohria would you mind adding the below `conflicts_with` stanza between the `homepage` and `app` stanzas:
```
  conflicts_with cask: [
                         'chromium',
                         'eloston-chromium',
                       ]
```

**Comment by fohria** [comment]
ah, cask-repair was not smart enough to catch that :) hopefully works now!

File: Casks/freesmug-chromium.rb [hunk 1]
```diff
@@ -7,6 +7,11 @@
   appcast 'https://sourceforge.net/projects/osxportableapps/rss?path=/Chromium'
   name 'Chromium'
   homepage 'http://www.freesmug.org/chromium'
+
```
Comment (miccal) [inline Casks/freesmug-chromium.rb:10]: Thanks for adding the `conflicts_with` stanza @fohria, but there is an extra space on this line you need to remove.

Comment (fohria) [reply Casks/freesmug-chromium.rb]: woops!

**Pull request merged**